src/app/asset-page/artstor-viewer/artstor-viewer.component.ts

```diff
@@ -248,6 +260,35 @@ export class ArtstorViewerComponent implements OnInit, OnDestroy, AfterViewInit
         }
     }
 
+    /**
+     * OpenSeaDragon: Show reference strip for multi-view
+     */
+    private attemptShowReferenceStrip() : void {
```

Comment:
I think it's a good idea to separate out the code for handling the ref. strip based on the assetCompareCount